sds_data_manager/lambda_code/SDSCode/path_helper.py

```diff
@@ -58,14 +44,18 @@ def __init__(self, filename):
         Parameters
         ----------
         filename : str
-            Filename
+            Science data filename
         """
         self.filename = filename
-        self.filename_convention = (
+        self.split_filename = self.filename.split("_")
```

Comment:
Not really critical at all, but do we need this as an attribute on self here or could it just be a local variable that you use below?
```suggestion
        split_filename = self.filename.split("_")
```
The same goes for the `last_value` below and wondering if that could also be a local variable instead.